settings.gradle

```diff
@@ -1 +1 @@
-include ':app', ':flickr', ':either'
+include ':app'
```

Comment:
Oops!
I just updated the master. 

Thanks for the PR @cdflynn , I should have waited for a min :)